teamcity.sh

```diff
@@ -1,6 +1,20 @@
 #!/bin/bash
 set -e
 
+setupNvm() {
```

Comment:
Hello, whats this...

Reply:
TeamCity has build steps when it runs builds.

Historically, with node applications, there has a step which installs nvm and runs `nvm use`, but it uses an old version of nvm which is not actively maintained. So we've been moving away from that in most cases to either using a CI script or running a docker container.

In this case, I've actually swapped out the docker container for a CI script, to (a) keep it in line with the other lambdas we maintain, (b) make it explicit where the node version is set for those reading through the repo and (c) because I'm not sure if there is a node 14 compatible docker container set up in TeamCity.

Reply:
See https://github.com/search?q=org%3Aguardian+nvm+use&type=code for other CI scripts using `nvm use`. 

Sometimes nvm needs some extra steps to boot it up, hence:
```
export NVM_DIR="$HOME/.nvm"
    [[ -s "$NVM_DIR/nvm.sh" ]] && . "$NVM_DIR/nvm.sh"  # This loads nvm
```

Reply:
I guess it needn't be a function call like that because it's unconditional, but it'll work